TINY-11877: Changelog 7.7.1 (#10211)
tinymce/tinymce: .changes/tinymce/7.7.1.md

## 7.7.1 - 2025-03-05

### Fixed
- Skin UI content CSS was truncated when bundling, causing CSS styles to be missing. #TINY-11875

> - Context forms used to disappear if their input were disabled on `onSetup`. #TINY-11890

Reviewer: ```suggestion
- Context forms would disappear if their input was disabled when `onSetup` was fired. #TINY-11890
```

Does it sound better?

Reviewer 2: I think the original entry isn't too bad. s/were/was and s/on/during would be better

> - Context forms used to disappear if their input were disabled on `onSetup`. #TINY-11890

Reviewer 2: ```suggestion
- Context forms used to disappear if their input was disabled in the `onSetup` API. #TINY-11890
```
